website/docs/mutation.md

```diff
@@ -0,0 +1,284 @@
+---
+id: mutation
+title: Mutation
+---
+
+#Gatekeeper mutation
+
+The mutation feature allows Gatekeeper to not only validate created Kubernetes resources but also modify them based on defined mutation policies.
+The feature is still in an alpha stage, so the final form can still change.
+
+Status: alpha
+
+## Mutation CRDs
+
+The mutation policies are defined by means of mutation specific CRDs:
+- AssignMetadata - defines changes to the metadata section of a resource
+- Assign - any change outside the metadata section
+
+The rules of mutating the metadata section are more strict than for mutating the rest of the resource definition. The differences will be described in more detail below.
+
+Here is an example of a simple AssignMetadata CRD:
+```yaml
+apiVersion: mutations.gatekeeper.sh/v1alpha1
+kind: AssignMetadata
+metadata:
+  name: demo-annotation-owner 
+spec:
+    match:
+      scope: Namespaced
+      kinds:
+        - apiGroups: ["*"]
+          kinds: ["Pod"]
+    location: "metadata.annotations.owner"
+    parameters: 
+      assign: 
+        value:  "admin"
+```
+
+Each mutation CRD can be divided into 3 distinct sections:
+- scope - what is to be modified (kinds, namespaces, ...)
```

Comment:
nit: this name collides with the `scope: Namespaced` match parameter above.

"selector" maybe? "binding"?

Reply:
I went with: "Extent of changes"
Selector - could be confused with the namespace selector
binding brings role binding to mind